src/auth/authenticator/api-key/config.go

```diff
@@ -0,0 +1,25 @@
+package apikey
+
+import (
+	"encoding/base64"
+	"hash"
+)
+
+type Config struct {
+	APIKeyFile map[string]*UserNameAndGroups
```

Comment:
why are you using pointer to that struct instead of the struct itself?

Reply:
I don't know. I tend to mostly use pointers to structs in maps. UserNameAndGroups is not that big, a habit of mine I would say :-)

Reply:
but you won't safe any memory or anything since the map is filled only in case elements a fullfilled, not the same than having struct with properties which can be empty where in that case you woud safe space

Reply:
Well actually I suppose it would save some space when building the Authenticator in case we have a huge csv file with many keys and long rows. A map of pointers has a smaller mem foot print than a map of structs.